pkg/cli/cmds/server.go

```diff
@@ -32,6 +32,11 @@ var (
 			Usage:  "(components) Enable rke2 default cloud controller manager's service controller",
 			EnvVar: "RKE2_ENABLE_SERVICELB",
 		},
+		&cli.BoolTFlag{
+			Name:   "cluster-init",
```

Comment:
While the wording comes over from K3s, I don't thing that there is any reason for use to reuse a confusing name as `cluster-init`. Its even more confusing that you need to "set it to false" for something to happen. Something like `sqlite` or `single-node-sqlite` could be clearer about what we intend.

Reply:
After more discussing, I've come around to just keeping it as is. The only people who will actually want/use this feature are likely coming from K3s anyways.

Reply:
@dereknola I think it is still a valid discussion, it isn't a very straightforward cli flag to be fair.

I think we should stick with the `single-node-sqlite`